app/views/catalog/_show_sidebar.html.erb

```diff
@@ -1,12 +1,2 @@
 <%= render 'show_tools' %>
-
-<% unless document.more_like_this.empty? %>
-  <div class="card">
-    <div class="card-header">More Like This</div>
-    <div class="card-body">
-      <%= render collection: document.more_like_this,
-                 partial: 'show_more_like_this',
```

Comment:
Do we leave this partial in for backwards compatibility?

Reply:
Exactly. I suspect it's not long for the world, but might make it into 8.x.